swig/cpp/src/Sysrepo.h

```diff
@@ -82,6 +82,13 @@
     #define S_Callback         std::shared_ptr<Callback>
 #endif
 
+#define SESS_DEFAULT 0
```

Comment:
Well, my biggest gripe with this is that this is not type-safe. I understand your point about SWIG generators and your approach where C++ is used mainly as a vehicle for other OO-oriented languages.

I'll be happy to help in designing something which works well under both SWIG and C++ once my copany-internal requests make it besides $legal, you know.